apps/docs-app/docs/features/testing/overview.md

```diff
@@ -18,6 +18,30 @@ Vitest supports many features:
 - In-source testing.
 - And more ...
 
+You can also [add Vitest](/docs/features/testing/vitest) to your existing project.
+
+## Angular support for Vitest
+
+In Angular v21, stable support for Vitest directly through the Angular CLI was introduced for new Angular projects. While both Analog and Angular support running tests with Vitest, there are some similarities and key differences.
+
+The table below shows the features available across both choices.
+
+| Vitest             | Analog    | Angular      |
+| ------------------ | --------- | ------------ |
+| Angular Versions   | v17+      | v21+         |
+| Support            | Community | Angular Team |
+| Builders           | ✅        | ✅           |
+| Schematics         | ✅        | ✅           |
+| Migrations         | ✅        | ✅           |
+| Fully Configurable | ✅        | ⚠️           |
+| Vitest CLI         | ✅        | ❌           |
+| Vitest Workpsaces  | ✅        | ❌           |
+| IDE extensions     | ✅        | ❌           |
+| Buildable Libs     | ✅        | ❌           |
+| Plugins            | ✅        | ❌           |
```

Comment:
I'd add:
- module "mocking" is broken: https://github.com/angular/angular-cli/issues/31609
- module graph is broken for the same reasons
- `happy-dom` is not supported yet (or any custom env): https://github.com/angular/angular-cli/issues/31653
- `preview` provider is not supported: https://github.com/angular/angular-cli/issues/31654
- types are not augmented: https://github.com/angular/angular-cli/issues/31656